src/components/popups/popup_connect.gd

```diff
@@ -32,13 +29,8 @@ func _on_ButtonConnect_pressed() -> void:
 
 
 func _on_ConnectPopup_about_to_show() -> void:
-	psk_line_edit.text = API.psk
 	var protocol:= "https://" if API.use_ssl else "http://"
-	adress_line_edit.text = protocol + API.adress + ":" + str(API.port)
-	$Content/Margin/Connect/PSK.show()
-	$Content/Margin/Connect/Adress.show()
-	connect_button.show()
-	status.text = "Resoto Core connection settings."
+	address_line_edit.text = protocol + API.adress + ":" + str(API.port)
```

Comment:
`API.adress` should likely also be named `.address`

Reply:
yeah, this is a typo since the very beginning, i need to change this in many places so i was planning to do all at once later..